app/views/application/_footer.html.erb

```diff
@@ -6,6 +6,8 @@
       <li><%= link_to 'Start a petition', check_petitions_path %></li>
       <li><%= link_to 'How petitions work', help_path %></li>
       <li><%= link_to 'Privacy and cookies', privacy_path %></li>
```

Comment:
Do we need to remove 'and cookies' here?

Reply:
My PR does that... so if we merged this first that would be covered

Reply:
Ah, so it does 👍🏻